### PR TITLE
Add contributing guidelines for external use

### PR DESCRIPTION
**Pull request by ankur22**
I noticed we didn't have any guidelines on how to contribute after I saw a comment on one of the issues.

These guidelines will guide external contributors on how to contribute to the project. It also adds a makefile to make it easier for external contributors to ensure their changes meet a set standard.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+Continuous integration will catch all of this if you don't, and it's fine to just fix linter complaints with another commit, but you can also run the linter yourself:
+
+```bash
+make check
+```
+
+Comments in the source should wrap at 100 characters, but there's no maximum length or need to be brief here - please include anything one might need to know in order to understand the code, that you could reasonably expect any reader to not already know (you probably don't need to explain what a goroutine is).
+
+#### Commit format
```
Comment (ankur22) [inline CONTRIBUTING.md:88]: This section may look familiar 😉

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# Contributing to xk6-browser
```
Comment (ankur22) [inline CONTRIBUTING.md:1]: It's pretty much a copy and paste from k6, with k6 replaced with xk6-browser in the appropriate places.

**Review by andrewslotin — APPROVED** [review]
👍

**Review by ka3de — COMMENTED** [review]
Overall LGTM, just made two minor comments to take into account, but not specific change requested.

**Review by inancgumus — APPROVED** [review]
This much resembles our internal k6-browser docs or am I having a Deja Vu? 😅

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+Thank you for your interest in contributing to xk6-browser!
+
+(ﾉ◕ヮ◕)ﾉ*:・ﾟ✧
+
+Before you begin, make sure to familiarize yourself with the [Code of Conduct](CODE_OF_CONDUCT.md). If you've previously contributed to other open source project, you may recognize it as the classic [Contributor Covenant](https://contributor-covenant.org/).
+
+If you want to chat with the team or the community, you can [join our community forums](https://community.k6.io/).
```
[the remark this answers is not in the document]
Comment (ankur22) [reply CONTRIBUTING.md]: Good suggestion. I've added it to a fixup 4a1ce4a7733865d4bd9962b236389c6a364c7467

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+Don't be afraid to file issues! Nobody can fix a bug we don't know exists, or add a feature we didn't think of.
+
+The worst that can happen is that someone closes it and points you in the right direction.
+
+That said, "how do I..."-type questions are often more suited for community forums.
+
+## Contributing code
+
+If you'd like to contribute code to xk6-browser, this is the basic procedure. Make sure to follow the [style guide](#style-guide) described below.
```
[the remark this answers is not in the document]
Comment (ankur22) [reply CONTRIBUTING.md]: I've added it to a fixup 4a1ce4a7733865d4bd9962b236389c6a364c7467

**Review by ka3de — APPROVED** [review]
Thanks for taking some time to do this!
LGTM.

**Pull request merged**